src/org/labkey/test/tests/AttachmentFieldTest.java

```diff
@@ -114,13 +114,13 @@ public void testAttachmentFieldInLists()
         clickAndWait(Locator.linkWithText(listName));
         DataRegionTable listTable = new DataRegionTable("query", getDriver());
         listTable.clickInsertNewRow();
-        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_PDF);
+        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_FILE);
         clickButton("Submit");
 
         log("Verify file opened in browser");
-        Locator.linkWithText(SAMPLE_PDF.getName()).findElement(getDriver()).click();
+        Locator.tagWithAttributeContaining("img", "title", SAMPLE_FILE.getName()).findElement(getDriver()).click();
         switchToWindow(1);
-        checker().verifyTrue("Incorrect PDF file displayed", getCurrentRelativeURL().contains(SAMPLE_PDF.getName()));
+        Assert.assertTrue("Incorrect file displayed", getCurrentRelativeURL().contains(SAMPLE_FILE.getName()));
```

Comment:
Use an assert that will show the actual value if this fails:
```suggestion
        Assertions.assertThat(getDriver().getCurrentUrl()).as("Incorrect file displayed").contains(SAMPLE_FILE.getName());
```